kstreams/middleware/udf_middleware.py

```diff
@@ -58,7 +67,8 @@ async def consume(cr: ConsumerRecord):
                 async def consume(cr: ConsumerRecord, stream: Stream):
                     ...
         """
-        params = self.bind_udf_params(cr)
+        self.bind_cr(cr)
+        params = [self.annotations_to_params[param_type] for param_type in self.params]
```

Comment:
creating the list here is actually faster than in a function 🤷🏻

Reply:
It makes sense because another object is called.